### PR TITLE
add: daily reminders

### PR DESCRIPTION
**Pull request by Smelton01**

### CODE REVIEW

**Comment by Smelton01** [comment]
#### Solution for issue #7 

#### Description  

Added a feature to the bot for taking a task description from the user and sending reminders every hour.

#### Tasks

- [x] Takes input from the user about what they want to accomplish in the entire day.
- [x] Sends a reminder after 60 minutes to see how far they have come up with the task.
- [x] After the end of the day asks the user if they have completed the task.
- [x] The bot will stop sending notifications once the user tells that they have finished the work.

File: bot.py [hunk 1]
```diff
@@ -64,12 +64,36 @@ async def stopwach(ctx):
...
+    await ctx.send(ctx.author.mention + f" Task: \"{task}\", should be completed today, \nHave done it already?")
+
+    # Check message source
+    def check(m):
+        return m.channel == ctx.channel and m.author == ctx.author
+
+    # Wait for user reply to cancel loop
+    try:
+        reply = await client.wait_for("message", check=check, timeout=300)
+        if reply.content.lower() in ["stop", "yes", "yep", "done", "sure"]:
```
Comment (mogiiee) [inline bot.py:88]: let the user know what commands can be used for yes and no

File: bot.py [hunk 2]
```diff
@@ -64,12 +64,36 @@ async def stopwach(ctx):
...
     # Takes input from the user (task) about what they would like to accomplish
-    # Send a reminder after one hour to see how far they have come up with the task
-    # At the end of every hour, ask the user if they have completed the task.
-    # Wait for the stop command from user (You may use "client.wait_for")
-    pass
+    task = " ".join(task)
+    await ctx.send(ctx.author.mention + f" Task: {task}, saved successfully")
+
+    # Start reminder loop
+    reminder_loop.start(ctx, task)
```
Comment (mogiiee) [inline bot.py:73]: add a function to let the user stop the remainders and add a new reminder all together

**Review by mogiiee — CHANGES_REQUESTED** [review]
great pr, just add a function to break the loop

**Pull request merged**